docs/dev-manuals/tooling/techstack-and-service-creation.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 3
+---
+
+import CustomImage from "@site/src/components/CustomImage";
+
+# Techstack and service creation
+
+### Techstack pverview
+
+This is a general overview of some of the technologies used in MiSArch and which services / components they involve.
+
+<CustomImage path="/images/techstack-overview" width="2803" height="2341" />
+
+A user client sends a request to Nginx to retrieve the frontend. The frontend then communicates with the backend through Nginx. Nginx acts both as a web-server and reverse proxy. The frontend creates requests in GraphQL. The GraphQL gateway resolves the request by querying the respective services.
+
+Dapr acts as a middleware abstracting communication between services. This enables metric and trace collection via Open Telemetry. Services communicate via Dapr sidecars. Each sidecar is associated with one service and known by the Dapr placement / master service, also known as Dapr Operator Service. The Dapr Operator feeds the Open Telemetry collector with metrics and traces.
+
+A full Zipkin stack stores and displays traces, while metrics are collected by Grafana / Prometheus. Each Service involved in GraphQL queries needs to expose a GraphQL compatible API. Each services needs to be registered in the GraphQL gateway.
```

Comment:
- each service involved in GraphQL queries needs to expose a GraphQL federation compatible API
- no, only services involved in GraphQL queries need to be registered in the gateway